nixos/modules/services/monitoring/scrutiny-collector.nix

```diff
@@ -0,0 +1,117 @@
+{ config, lib, pkgs, ... }:
+
+let
+  inherit (lib) mkEnableOption mkIf mkOption mkPackageOption types mdDoc;
+
+  cfg = config.services.scrutiny-collector;
+
+  prettyJSON = conf:
+    pkgs.runCommand "loki-config.json" { } ''
+      echo '${builtins.toJSON conf}' | ${pkgs.jq}/bin/jq 'del(._module)' > $out
+    '';
+
+in
+{
+  options.services.scrutiny-collector = {
+    enable = mkEnableOption "scrutiny-collector";
+
+    package = mkPackageOption pkgs "scrutiny-collector" { };
+
+    api-endpoint = mkOption {
+      type = types.str;
+      default = "http://localhost:8080";
+      description = mdDoc ''
+        The api server endpoint
+      '';
+    };
+
+    log-file = mkOption {
+      type = types.nullOr types.path;
```

Comment:
I'm not sure if `types.path` is the correct type to use here. Would `string` be preferable?